README.md

```diff
@@ -6,6 +6,14 @@ The Chef Management Console is a proprietary addon for the Chef Server. This rep
 
 If you need to file an issue against another Chef project, you can find a list of projects and where to file issues in the [community contributions section](https://docs.chef.io/community_contributions.html#issues-and-bug-reports) of the [Chef docs](https://docs.chef.io).
 
+# Changelog
```

Comment:
You didn't read the commit messages did you? ;) I put the changelog file in place for people who don't read the readme and go straight there, mostly to fulfill the convention. I suppose we could drop the line from the readme.